master/src/main/proto/Resource.proto

```diff
@@ -22,6 +22,7 @@ package deploy.master.meta;
 
 enum Type {
   UnRegisterShuffle = 12;
+  BatchUnRegisterShuffle = 28;
```

Comment:
Place this new type to the end of this section.

Reply:
Done!